test/props/prop-set.js

```diff
@@ -112,6 +112,9 @@ describe('PropSet', () => {
       assert.strictEqual(set.def.props.g.value, 'green')
       assert.strictEqual(set.def.props.h.value, 'green')
     })
+    it('resolves aliases set inside of alias objects', () => {
+      assert.strictEqual(set.def.props.j.value, 'green')
```

Comment:
And what would be a way to make a lookup for `otherInfo`? Should it simply be ignored or should it land as part of the props?